version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.5.0-SNAPSHOT"
+version in ThisBuild := "0.5.1"
```

Comment:
we should start just bumping version with every code change, because later it gets hard to keep track of what exact version the given jar is...

Reply:
That seems pretty error-prone.  Why not always use versions of spark-sql-perf produced as actual releases (with corresponding git tags)?  Is publishing too heavy weight a process?

Reply:
Over the last few months we kept merging various small changes, including ones that changed some interfaces (all around TPCDS benchmark) without changing the version number. So now there are various jars circulating that are hard to distinguish what code version they are precisely. Most of this mess is my fault, apologies.
I figured out that maybe bumping the minor version with every code change made is a good idea, but I have no idea what are the actual best practices of doing that re: git tags / branches, version dependencies for maven etc. etc.
So for this change, since it's unrelated, let's just keep the version and push it that is.

Going forward @npoggi will be managing spark-sql-perf so I'll leave to him figuring out what is the best practice way to version and release it :-).